docs/manuals/core/versioning/experiment_mode.md

```diff
@@ -0,0 +1,197 @@
+
+```console
+$ python main.py -l
+Version number                         Mode                   Creation date
+26e56e84-9e7e-4a26-93f6-443d9aa541d9   Development (latest)   2023-01-25 12:20:33
+
+$ python main.py --experiment
+[2023-01-25 12:20:56,474][Taipy][INFO] job JOB_my_print_algo_e1c49bdb-9284-40c5-a096-db0235697cb3 is completed.
+nb scenarios: 1
+
+$ python main.py -l
+Version number                         Mode                   Creation date
+325d0618-6f9e-459b-9597-48fa93a57a23   Experiment (latest)    2023-01-25 12:20:56
+26e56e84-9e7e-4a26-93f6-443d9aa541d9   Development            2023-01-25 12:20:33
+```
+
+An _experiment_ version is created and stored. As you can see the number of scenarios
+displayed is 1. That means Taipy only considers the scenarios of the _experiment_
+version, and filters out the entities of the other development version. By default,
+a random name is used to identify the version of your application.
+
+# Name a version
+
+You can explicitly define the name of an _experiment_ version by providing the name
+in the command line.
+
+```console
+$ python main.py -l
+Version number                         Mode                   Creation date
+325d0618-6f9e-459b-9597-48fa93a57a23   Experiment (latest)    2023-01-25 12:20:56
+26e56e84-9e7e-4a26-93f6-443d9aa541d9   Development            2023-01-25 12:20:33
+
+$ python main.py --experiment 1.0
+[2023-01-25 12:24:19,613][Taipy][INFO] job JOB_my_print_algo_9b6232f9-601e-4a85-852e-2ada7bc1e459 is completed.
+nb scenarios: 1
+
+$ python main.py -l
+Version number                         Mode                   Creation date
+1.0                                    Experiment (latest)    2023-01-25 12:24:19
+325d0618-6f9e-459b-9597-48fa93a57a23   Experiment             2023-01-25 12:20:56
+26e56e84-9e7e-4a26-93f6-443d9aa541d9   Development            2023-01-25 12:20:33
+```
+
+With `--experiment 1.0` option, an _experiment_ version is created and stored under
+the name "1.0".
+
+In this example, you can see the number of scenarios displayed is still 1. Taipy
+only considered the scenarios of the 1.0 version, and filters out the entities of
+the other versions.
+
+# Run an existing version
+
+To run an existing _experiment_ version, the configuration of the application must
+be the same. No change must be done. You can run a Taipy application on your
+command line interface with `--experiment VERSION` option and the name of the existing version.
+
+```console
+$ python main.py -l
+Version number                         Mode                   Creation date
+1.0                                    Experiment (latest)    2023-01-25 12:24:19
+325d0618-6f9e-459b-9597-48fa93a57a23   Experiment             2023-01-25 12:20:56
+26e56e84-9e7e-4a26-93f6-443d9aa541d9   Development            2023-01-25 12:20:33
+
+$ python main.py --experiment 1.0
+[2023-01-25 12:28:54,963][Taipy][INFO] job JOB_my_print_algo_2133dc18-643b-4351-872b-aedfc2c65c9c is completed.
+nb scenarios: 2
```

Comment:
When someone runs the code example that we provide, the number of scenarios is not shown on cli right? 

Can we have a better way to demonstrate this?

Reply:
Yes it is. I added a print in the main.py file.

Reply:
Forgive me if I miss it, but I can't find the print command in the code example that we provide

Reply:
OH right That s my bad. I forgot to update python files...